signing_clients/apps.py

```diff
@@ -357,41 +366,19 @@ def make_signed(self, signature, outpath=None, sigpath=None):
                     zout.writestr('META-INF/ids.json', self.ids)
 
 
-class JarSigner(object):
```

Comment:
This was unused in the whole package so I removed it, I also could not find any reference to it in either AMO code or anywhere else on GitHub. Please let me know if you know if any usage that requires it to be kept.